paasta_tools/utils.py

```diff
@@ -659,6 +715,40 @@ def get_fsm_deploy_pipeline(self):
             raise PaastaNotConfiguredError(
                 'Could not find fsm_deploy_pipeline in configuration directory: %s' % self.directory)
 
+    def get_log_writer(self):
+        """Get the log_writer configuration out of global paasta config
+
+        :returns: The log_writer dictionary.
+        """
+        try:
+            return self['log_writer']
+        except KeyError:
+            raise PaastaNotConfiguredError('Could not find log_writer in configuration directory: %s' % self.directory)
```

Comment:
Also for actual deployments, I think I'd like log_writer to be required. There's no sane default: if we default to `null` or a hypothetical `stderr` writer, then `paasta logs` would not work, which is the whole point of this. There's a separate subsystem for writing to stderr: python's logging.

If someone wants to deploy paasta without paasta logs working, they can set up the `null` log_writer.